app/controllers/healthcheck_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HealthcheckController < ApplicationController
+  skip_after_action :verify_authorized
+
+  def api_tokens
+    render json: Healthcheck::ApiTokens.new.to_hash
```

Comment:
🤔 actually how does `.new` work here. I can't see an initalizer in the class and it doesn't inherit?
https://github.com/alphagov/signon/blob/master/lib/healthcheck/api_tokens.rb

Reply:
It just doesn't have a constructor

Reply:
cool cool, just tested it and all looks good - rare I see a class that doesn't have any parameters.
I think more often i've seen this sort of thing done with a

```ruby
module Foo
  class << self
    def call; end
  end
end
```

sort of thing